test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -51,11 +51,11 @@ estimate: 90m
     oc get prometheusrules -n redhat-rhoam-marin3r
     ```
 
-3.  Modify the `sku-limits-managed-api-service` configmap to set the rate limit to 100 requests per minute.
```

Comment:
Hi @jackdelahunt this one falls under the category of staying the same because it belongs to the old implementation.   This test case requires an update as part of -> https://issues.redhat.com/browse/MGDAPI-1609 so we don't need to make any changes in this file for now.